test/basic_test.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+import time
+
+from test.lib.user import TestUser
+from test.lib.test_server import TestServer
+
+from server.models import init_db
+from server.models.utils import create_users, make_friends
+from server.base_client import USER_STATUS_DEFAULT, CLIENT_OFFLINE
+from server.config import PROJECT_PATH
+
+from protocol.messages import NormalMessage, CMD_CHANGE_STATUS
+
+
+class TestFunctional(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.server = TestServer()
+        init_db(db=cls.server.config.db, enable_logging=False)
+        cls.name_pass1 = ('user1', 'pass1')
+        cls.name_pass2 = ('user2', 'pass2')
+        cls.name_pass3 = ('user3', 'pass3')
+        cls.users = create_users(name_pass=[cls.name_pass1, cls.name_pass2, cls.name_pass3, ])
+        make_friends(cls.users[cls.name_pass1[0]], [cls.users[cls.name_pass2[0]], cls.users[cls.name_pass3[0]]])
+
+    def setUp(self):
+        self.server.up()
+
+    def tearDown(self):
```

Comment:
Mix of cases. Code style.

Reply:
Can't do nothing with that - it is from unittest lib naming